plugin/src/lib.rs

```diff
@@ -397,10 +437,20 @@ impl<'a, 'cx> MutatorPlugin<'a, 'cx> {
                     ],
                 );
                 quote_expr!(self.cx(), {
-                    ::mutagen::report_coverage($n...$current);
+                    ::mutagen::report_coverage($n..$current, &$sym[$flag], $mask);
                     ::mutagen::ge($right, $left, $n)
                 })
             }
+            BinOpKind::Add => {
```

Comment:
Adding this new mutation is intentional?

Reply:
Yes, I wanted to see if it works with opportunistic mutations. I'm going to extend the build script to generate code for all of them to be included in this place next.